sda-commons-server-key-mgmt/src/main/java/org/sdase/commons/keymgmt/model/ValueMappingModel.java

```diff
@@ -0,0 +1,44 @@
+package org.sdase.commons.keymgmt.model;
+
+import java.util.*;
+
+@SuppressWarnings("unused")
+public class ValueMappingModel {
+
+  private final Map<String, String> apiToImplBidirectional = new HashMap<>();
+  private final Map<String, String> implToApiBidirectional = new HashMap<>();
+
+  private final Map<String, String> implToApi = new HashMap<>();
+  private final Map<String, String> apiToImpl = new HashMap<>();
```

Comment:
Some documentation concerning the priority of the latter maps in comparison to the "bidirectional" maps would be helpful.

Reply:
ok